ui/app/templates/clients/client.hbs

```diff
@@ -1,13 +1,131 @@
+      <div data-test-drain-updated-notification class="columns">
+        <div class="column">
+          <h3 data-test-title class="title is-4">Drain Updated</h3>
+          <p data-test-message>The new drain specification has been applied.</p>
+        </div>
+        <div class="column is-centered is-minimum">
+          <button data-test-dismiss class="button is-info" onclick={{action (mut showDrainUpdateNotification) false}}>Okay</button>
+        </div>
+      </div>
+    </div>
+  {{/if}}
+  {{#if showDrainNotification}}
+    <div class="notification is-info">
+      <div data-test-drain-complete-notification class="columns">
+        <div class="column">
+          <h3 data-test-title class="title is-4">Drain Complete</h3>
+          <p data-test-message>Allocations have been drained and the node has been set to ineligible.</p>
+        </div>
+        <div class="column is-centered is-minimum">
+          <button data-test-dimiss class="button is-info" onclick={{action (mut showDrainNotification) false}}>Okay</button>
+        </div>
+      </div>
+    </div>
+  {{/if}}
+  <div class="toolbar">
+    <div class="toolbar-item is-top-aligned is-minimum">
+      <span class="title">
+        <span data-test-node-status="{{model.compositeStatus}}" class="node-status-light {{model.compositeStatus}}"></span>
+      </span>
+    </div>
+    <div class="toolbar-item">
+      <h1 data-test-title class="title with-subheading">
+        {{or model.name model.shortId}}
+      </h1>
+      <p>
+        <label class="is-interactive">
+          {{#toggle
+            data-test-eligibility-toggle
+            isActive=model.isEligible
+            isDisabled=(or setEligibility.isRunning model.isDraining)
+            onToggle=(perform setEligibility (not model.isEligible))}}
+            Eligible
+          {{/toggle}}
+          <span class="tooltip" aria-label="Only eligible clients can receive allocations">
+            {{x-icon "info-circle-outline" class="is-faded"}}
+          </span>
+        </label>
+        <span data-test-node-id class="tag is-hollow is-small no-text-transform">
+          {{model.id}}
+          {{copy-button clipboardText=model.id}}
+        </span>
+      </p>
+    </div>
+    <div class="toolbar-item is-right-aligned is-top-aligned">
+      {{#if model.isDraining}}
+        {{two-step-button
+          data-test-drain-stop
+          idleText="Stop Drain"
+          cancelText="Cancel"
+          confirmText="Yes, Stop"
+          confirmationMessage="Are you sure you want to stop this drain?"
+          awaitingConfirmation=stopDrain.isRunning
+          onConfirm=(perform stopDrain)}}
+      {{/if}}
+    </div>
+    <div class="toolbar-item is-right-aligned is-top-aligned">
+      {{drain-popover
+        client=model
+        onDrain=(action "drainNotify")
+        onError=(action "drainError")}}
+    </div>
+  </div>
 
   <div class="boxed-section is-small">
     <div class="boxed-section-body inline-definitions">
```

Comment:
Can we remove the Draining and Eligibility status from here now that the UI accounts for it via the eligibility toggle and the drain strategy component? Not sure this is the right spot to comment but wanted to make sure you saw this.